### DISCUSSION TITLE
Welcome to vCardEditor Discussions!

### DISCUSSION THREAD

**Discussion by abdelkader** [abdelkader/vCardEditor#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rhinoroo** [comment]
I really appreciate your project, as it is one of the few I have found that suits my needs for managing a single large VCF file that I continually need to maintain and share.  THANK YOU!

The search works great on the selection list but I would love to see some sorting options as well as additional column selections if possible!

**Comment by abdelkader** [comment]
Thanks for the kind words!

I'll add those features to the backlog..

**Comment by abdelkader** [comment]
These feature are now implemented! I'll make a new release so you can test.

**Comment by marcinBB8** [comment]
How run it on win10?
I dont see exe file

**Reply by abdelkader** [reply]
from the relase section : https://github.com/abdelkader/vCardEditor/releases

**Comment by jameswintermute** [comment]
Hi Abelkader, I was interested in your project but on downloading it appears to be a portable executable for Microsoft platforms. You should put that in the readme that it is for windows and not *nix if that is the case.

Thanks

**Reply by abdelkader** [reply]
I ported this application now to wails, soon it will be available to other platforms

https://github.com/abdelkader/Kardoo
